src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala

```diff
@@ -696,28 +696,50 @@ object MobilitySimulator
       config.mobsim.input.mobility.source.colSep
     )
 
-    val evs = ElectricVehicle.createEvs(
-      numberOfEvsInArea,
-      poisWithSizes
-        .getOrElse(
-          CategoricalLocationDictionary.HOME,
-          throw InitializationException(
-            "Unable to obtain the probability density function for home POI."
-          )
-        )
-        .pdf,
-      poisWithSizes.getOrElse(
-        CategoricalLocationDictionary.WORK,
+    val homePOIsWithSizes = poisWithSizes
+      .getOrElse(
+        CategoricalLocationDictionary.HOME,
         throw InitializationException(
-          "Unable to obtain the probability density function for work POI."
+          "Unable to obtain the probability density function for home POI."
         )
-      ),
-      chargingStations,
-      startTime,
-      targetShareOfHomeCharging,
-      evModelPdf,
-      tripProbabilities.firstDepartureOfDay
+      )
+      .pdf
+
+    val workPoisWithSizes = poisWithSizes.getOrElse(
+      CategoricalLocationDictionary.WORK,
+      throw InitializationException(
+        "Unable to obtain the probability density function for work POI."
+      )
```

Comment:
Shouldn't this be handled the same way as ´homePOIsWithSizes´ => with .pdf ?

Reply:
We split the home pois later on with regards of whether they can charge at home or not. Therefore we extract it from the original `ProbabilityDensityFunction` and build it for both possibilities again. With the work pdf we don't do such a thing.